new-integration-tests/src/test/java/oracle/weblogic/domain/AdminServer.java

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.domain;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+@ApiModel(description = "AdminServer represents the operator configuration for the Administration Server.")
+public class AdminServer {
+
+  @ApiModelProperty(
+      "Configures which of the Administration Server's WebLogic admin channels should be exposed outside"
+          + " the Kubernetes cluster via a node port service.")
+  private AdminService adminService;
+
+  @ApiModelProperty(
+      "The strategy for deciding whether to start a server. "
+          + "Legal values are ALWAYS, NEVER, or IF_NEEDED.")
+  private String serverStartPolicy;
+
+  @ApiModelProperty("Configuration affecting server pods.")
+  private ServerPod serverPod;
+
+  @ApiModelProperty(
+      "Customization affecting ClusterIP Kubernetes services for WebLogic Server instances.")
+  private ServerService serverService;
+
+  @ApiModelProperty(
+      "The state in which the server is to be started. Use ADMIN if server should start "
+          + "in the admin state. Defaults to RUNNING.")
+  private String serverStartState;
+
+  @ApiModelProperty(
+      "If present, every time this value is updated the operator will restart"
+          + " the required servers.")
+  private String restartVersion;
+
+  public AdminServer adminService(AdminService adminService) {
+    this.adminService = adminService;
+    return this;
+  }
+
+  public AdminService getAdminService() {
```

Comment:
Sorry if I am off here, but I am new to the fluent API approach.  From what I read, I don't think they use "get" or "set" for fluent API.  For example, given a field "pepper", the getter for pepper is just pepper(), and the setter is pepper(T newValue).  So in this case, adminService(AdminService x), and adminService().

Reply:
oh, I see... okay.  thanks.